<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>780</width>
    <height>664</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>780</width>
    <height>664</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>780</width>
    <height>664</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>串口通信 ©2015 wdLin @ USTC</string>
  </property>
  <property name="windowIcon">
   <iconset resource="icon.qrc">
    <normaloff>:/new/icon.ico</normaloff>:/new/icon.ico</iconset>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QGroupBox" name="serialPortSettingGroupBox">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>171</width>
      <height>221</height>
     </rect>
    </property>
    <property name="title">
     <string>串口设置</string>
    </property>
    <widget class="QLabel" name="staticSerialPortNumberLabel">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>30</y>
       <width>54</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>串口号：</string>
     </property>
    </widget>
    <widget class="QLabel" name="staticBaudRateLabel">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>60</y>
       <width>54</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>波特率：</string>
     </property>
    </widget>
    <widget class="QLabel" name="staticParityLabel">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>90</y>
       <width>54</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>校验位：</string>
     </property>
    </widget>
    <widget class="QLabel" name="staticDataBitsLabel">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>120</y>
       <width>54</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>数据位：</string>
     </property>
    </widget>
    <widget class="QLabel" name="staticStopBitsLabel">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>150</y>
       <width>54</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>停止位：</string>
     </property>
    </widget>
    <widget class="QPushButton" name="onOffSerialPortButton">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>182</y>
       <width>71</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>新宋体</family>
      </font>
     </property>
     <property name="text">
      <string>打开串口</string>
     </property>
    </widget>
    <widget class="QComboBox" name="baudRateComboBox">
     <property name="geometry">
      <rect>
       <x>70</x>
       <y>60</y>
       <width>69</width>
       <height>22</height>
      </rect>
     </property>
     <property name="currentIndex">
      <number>3</number>
     </property>
     <item>
      <property name="text">
       <string>1200</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>2400</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>4800</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>9600</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>19200</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>38400</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>57600</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>115200</string>
      </property>
     </item>
    </widget>
    <widget class="QComboBox" name="parityComboBox">
     <property name="geometry">
      <rect>
       <x>70</x>
       <y>90</y>
       <width>69</width>
       <height>22</height>
      </rect>
     </property>
     <item>
      <property name="text">
       <string>None</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Even</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Odd</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Space</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Mark</string>
      </property>
     </item>
    </widget>
    <widget class="QComboBox" name="dataBitsComboBox">
     <property name="geometry">
      <rect>
       <x>70</x>
       <y>120</y>
       <width>69</width>
       <height>22</height>
      </rect>
     </property>
     <property name="currentIndex">
      <number>3</number>
     </property>
     <item>
      <property name="text">
       <string>5</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>6</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>7</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>8</string>
      </property>
     </item>
    </widget>
    <widget class="QComboBox" name="stopBitsComboBox">
     <property name="geometry">
      <rect>
       <x>70</x>
       <y>150</y>
       <width>69</width>
       <height>22</height>
      </rect>
     </property>
     <item>
      <property name="text">
       <string>1</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>1.5</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>2</string>
      </property>
     </item>
    </widget>
    <widget class="QLineEdit" name="serialPortLineEdit">
     <property name="geometry">
      <rect>
       <x>70</x>
       <y>30</y>
       <width>91</width>
       <height>27</height>
      </rect>
     </property>
     <property name="text">
      <string>/dev/ttyS0</string>
     </property>
    </widget>
   </widget>
   <widget class="QTextBrowser" name="textBrowser">
    <property name="geometry">
     <rect>
      <x>195</x>
      <y>10</y>
      <width>571</width>
      <height>491</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>新宋体</family>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="contextMenuPolicy">
     <enum>Qt::NoContextMenu</enum>
    </property>
    <property name="html">
     <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'新宋体'; font-size:14pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot;-qt-paragraph-type:empty; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;br /&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
    <property name="acceptRichText">
     <bool>false</bool>
    </property>
    <property name="textInteractionFlags">
     <set>Qt::LinksAccessibleByMouse|Qt::TextSelectableByMouse</set>
    </property>
    <property name="openLinks">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="InputPlainTextEdit" name="inputPlainTextEdit">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>520</y>
      <width>451</width>
      <height>111</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>新宋体</family>
      <pointsize>14</pointsize>
     </font>
    </property>
   </widget>
   <widget class="QPushButton" name="sendMessageButton">
    <property name="geometry">
     <rect>
      <x>650</x>
      <y>580</y>
      <width>91</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>发送</string>
    </property>
   </widget>
   <widget class="QGroupBox" name="groupBox">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>250</y>
      <width>151</width>
      <height>111</height>
     </rect>
    </property>
    <property name="title">
     <string>功能</string>
    </property>
    <widget class="QPushButton" name="clearShowButton">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>20</y>
       <width>75</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>清空显示</string>
     </property>
    </widget>
    <widget class="QPushButton" name="clearInputButton">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>60</y>
       <width>75</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>清空输入</string>
     </property>
    </widget>
   </widget>
   <widget class="QCheckBox" name="enterSendCheckBox">
    <property name="geometry">
     <rect>
      <x>650</x>
      <y>520</y>
      <width>111</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Enter键发送</string>
    </property>
    <property name="checked">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QCheckBox" name="autoEnterCheckBox">
    <property name="geometry">
     <rect>
      <x>650</x>
      <y>550</y>
      <width>121</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>发送自动换行</string>
    </property>
    <property name="checked">
     <bool>true</bool>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>780</width>
     <height>28</height>
    </rect>
   </property>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>InputPlainTextEdit</class>
   <extends>QPlainTextEdit</extends>
   <header location="global">InputPlainTextEdit.h</header>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="icon.qrc"/>
 </resources>
 <connections/>
</ui>
